src/main/java/me/srgantmoomoo/postman/framework/module/Module.java

```diff
@@ -39,22 +38,22 @@ public Module(String name, String description, int key, Category category) {
 	
 	public void addSettings(Setting... settings) {
 		this.settings.addAll(Arrays.asList(settings));
-		this.settings.sort(Comparator.comparingInt(s -> s == keyCode ? 1 : 0));
+		this.settings.sort(Comparator.comparingInt(setting -> setting == this.keyCode ? 1 : 0));
 	}
 	
 	public String getDescription() {
-		return description;
+		return this.description;
```

Comment:
Ok this shit mad hard to do on my phone… anyways, why are you making everrything this.? The program already knows what is being referenced, theres i nothin else in this class using those names

Reply:
cause oop